### DISCUSSION TITLE
For AMP pages is there any way to use ?amp rather than ?amp=1

### DISCUSSION THREAD

**Discussion by cidicles** [vercel/next.js#22090] [Help]
I was wondering if there is any way to use `?amp` in the url rather than `?amp=1`.

**Comment by krystianjj** [comment]
In nextjs documentation you have `The page is rendered as traditional HTML (default) and AMP HTML (by adding ?amp=1 to the URL)`, because nextjs have own hook
`import { useAmp } from 'next/amp';`
to recognize amp version of your site.
In generally if you write your own function to recognize that is `amp` version you can use `?amp` in url.
